keyboards/pabile/p40/p40.h

```diff
@@ -0,0 +1,19 @@
+#ifndef P40_H
+#define P40_H
+
+#include "quantum.h"
+
+#define LAYOUT_ortho_4x10( \
+      k101, k102, k103, k104, k105, k106, k107, k108, k109, k110, \
+      k201, k202, k203, k204, k205, k206, k207, k208, k209, k210, \
+      k301, k302, k303, k304, k305, k306, k307, k308, k309, k310, \
+      k401, k402, k403, k404, k405, k406, k407, k408, k409, k410 \
+) \
+{ \
+      {k101, k102, k103, k104, k105, k106, k107, k108, k109, k110}, \
+      {k201, k202, k203, k204, k205, k206, k207, k208, k209, k210}, \
+      {k301, k302, k303, k304, k305, k306, k307, k308, k309, k310}, \
+      {k401, k402, k403, k404, k405, k406, k407, k408, k409, k410} \
```

Comment:
```suggestion
      k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, \
      k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, \
      k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, \
      k30, k31, k32, k33, k34, k35, k36, k37, k38, k39 \
) \
{ \
      {k00, k01, k02, k03, k04, k05, k06, k07, k08, k09}, \
      {k10, k11, k12, k13, k14, k15, k16, k17, k18, k19}, \
      {k20, k21, k22, k23, k24, k25, k26, k27, k28, k29}, \
      {k30, k31, k32, k33, k34, k35, k36, k37, k38, k39} \
```
Again I think hex numbering and 0-based row/col indexing is easier to follow and read.  I'm probably doing this the stupid way as there is surely a 4x10 community layout.